src/components/ArticlesCard.jsx

```diff
@@ -0,0 +1,65 @@
+import { Link } from "react-router-dom";
+
+function ArticlesCard({ article, setArticleID }) {
+  if (article.body) {
```

Comment:
I'm not really sure what this if/else is doing, I would expect all your articles to have a body. It's also not very DRY code